### PR TITLE
feat: :sparkles: create sdk package

### PR DESCRIPTION
**Pull request by Mario2206**
Create the sdk package. 

### How to test? 

First of all, you need to build the packages. To do so, in packages/app run the command `yarn package` and in packages/sdk, run `yarn build`.

I suggest you to init a new project and then install the sdk and the fsdt app in this new project (using yarn link). Then, you should create 2 files : one for the monitor and the second one for the source. Finally, you can instantiate the FsdtLogger in both files : the source one should send the log and the monitor one should receive the log.

Obviously, the monitor will receive the logs as soon as the aggregator server is started.
You can start it by running this command `yarn fullstack-devtool -p 1234`

**Source file:**
```typescript
import FsdtLogger from "@fullstack-devtool/sdk";

const logger = new FsdtLogger("test-app-1", {
	port: 1234,
	connectionType: "source",
});

setInterval(() => {
	logger.info({ myVar: "myVar" });
}, 1000);
``` 

**Monitor file:**
```typescript
import FsdtLogger from "@fullstack-devtool/sdk";

const logger = new FsdtLogger("test-app-1", {
	port: 1234,
	connectionType: "monitor",
});

logger.onLogReceived((message) => {
	console.log({ message });
});

setInterval(() => {}, 1 << 30);

```

### CODE REVIEW

File: packages/core/types/FsdtMessage.ts [hunk 1]
```diff
@@ -10,8 +11,9 @@ export type FsdtServerMessage = FsdtMessage & {
 }
 
 export type FsdtLogMessage = {
-  log: string;
+  content: string;
   timestamp: number;
```
Comment (Maxime-p) [inline packages/core/types/FsdtMessage.ts:15]: is timestamp the best way to store datetime ?

File: packages/core/types/FsdtMessage.ts [hunk 2]
```diff
@@ -10,8 +11,9 @@ export type FsdtServerMessage = FsdtMessage & {
 }
 
 export type FsdtLogMessage = {
```
Comment (Maxime-p) [inline packages/core/types/FsdtMessage.ts:13]: Missing source ?

File: packages/core/types/FsdtMessage.ts (hunk 1 above)
Comment (Mario2206) [reply packages/core/types/FsdtMessage.ts:15]: Do you have another idea?

File: packages/core/types/FsdtMessage.ts (hunk 2 above)
Comment (Mario2206) [reply packages/core/types/FsdtMessage.ts:13]: Only the FsdtServerMessage has the field "source"

File: packages/core/types/FsdtMessage.ts (hunk 1 above)
Comment (Maxime-p) [reply packages/core/types/FsdtMessage.ts:15]: Maybe UTC date/time, timestand isn't easely readable. Everywhere we use it, we gonna have to reformat it. I'm ok with timestanp, I'm just thinking if it's the best choice

File: packages/core/types/FsdtMessage.ts (hunk 2 above)
Comment (Maxime-p) [reply packages/core/types/FsdtMessage.ts:13]: Ok, I think I miss understood how it works, mb

File: packages/core/types/FsdtMessage.ts (hunk 1 above)
Comment (Mario2206) [reply packages/core/types/FsdtMessage.ts]: Ok I fixed it

**Review by Maxime-p — APPROVED** [review]

**Pull request merged**